packages/p2p/src/components/orders/orders-user-rating-button/orders-user-rating-button.scss

```diff
@@ -0,0 +1,7 @@
+.orders-user-rating-button {
+    margin-left: auto;
+
+    &-icon {
+        padding-right: 0.5rem;
```

Comment:
Probably best to use `margin- right` since padding messes up the icon usually

Reply:
ah yeah, there have been issues with the padding with the icon, will change it